android/src/main/kotlin/com/example/another_location_plugin/AnotherLocationPlugin.kt

```diff
@@ -36,35 +45,81 @@ class AnotherLocationPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
                 permissionEventsSource = null
             }
         }
-    private lateinit var context: Context
-    private lateinit var activity: Activity
 
+    private lateinit var fusedLocationProviderClient: FusedLocationProviderClient
+    private lateinit var locationRequest: LocationRequest
+    private lateinit var locationCallback: LocationCallback
+
+    private lateinit var locationEventsEmitter: EventChannel
+    private var locationEventsSource: EventChannel.EventSink? = null
```

Comment:
Avoid to assign null values, use lateinit instead to this

Reply:
Understand, Done!